crowbar_framework/lib/crowbar/repository.rb

```diff
@@ -105,6 +105,26 @@ def where(options = {})
         result
       end
 
+      def disable_by(options = {})
+        version = options.fetch(:version, nil)
+        return if version.nil?
+
+        all_platforms.each do |platform|
+          arches(platform).each do |arch|
+            repositories(platform, arch).each do |repo|
```

Comment:
I'm not sure it will work here: the issue is that Cloud 6 repos are not even returned in the list here; they're just in the data bag, but not in the yaml file.